app/src/main/cpp/skyline/loader/nsp.cpp

```diff
@@ -57,19 +57,16 @@ namespace skyline::loader {
         return NcaLoader::LoadExeFs(this, programNca->exeFs, process, state);
     }
 
-    std::vector<u8> NspLoader::GetIcon() {
-        if (romFs == nullptr)
+    std::vector<u8> NspLoader::GetIcon(languages::ApplicationLanguage language) {
+        if (controlRomFs == nullptr)
```

Comment:
@bylaws I would like confirmation whether this is correct or not (and if it was a mistake in the first place)